packages/rrweb-snapshot/test/rebuild.test.ts

```diff
@@ -47,6 +47,37 @@ describe('rebuild', function () {
     });
   });
 
+  describe('shadowDom', function () {
+    it('should rebuild shadowRoot with siblings', function () {
```

Comment:
The name of this test case is 'build shadowRoot with siblings' but actually only the shadowRoot itself is rebuilt without any siblings.

Reply:
Good catch @Mark-Fenng 
```suggestion
    it('rebuild shadowRoot without siblings', function () {
```